packages/compat/src/transaction.ts

```diff
@@ -86,8 +86,7 @@ function convertSignatures(
 }
 
 export function fromVersionedTransactionWithBlockhash(
-    transaction: VersionedTransaction,
-    lastValidBlockHeight: bigint
+    transaction: VersionedTransaction
 ): Transaction & ITransactionWithFeePayer & ITransactionWithBlockhashLifetime {
```

Comment:
Also you decided not to overload here? You are just going to default to `u64::MAX` no matter what?

Reply:
Updated to add an optional `lastValidBlockHeight` per the latest discussion!